src/System.Text.Primitives/System/Text/InvariantParser_signed.tt

```diff
@@ -0,0 +1,321 @@
+﻿<#@ template hostspecific="false" language="C#" #>
+<#@ output extension=".cs" #>
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
I assume this template file is used to generate the cs files. could you please add a comment here telling don't modify this file manually and instead modify the template file .tt